api/src/integrations/bcogc/inspections-utils.js

```diff
@@ -41,7 +42,15 @@ class Inspections extends BaseRecordUtils {
     inspection['_sourceRefOgcDeficiencyId'] = csvRow['deficiency objectid'] || null;
 
     inspection['recordType'] = 'Inspection';
-    inspection['dateIssued'] = csvRow['inspection date'] || null;
+    let inspectionDate = null;
```

Comment:
Could just do `inspection['dateIssued'] = csvRow['inspection date'] ? moment.tz(csvRow['inspection date'], "DD-MMM-YYYY", "America/Vancouver").toDate() : null;` here to make it simpler

Reply:
I had that originally! But it didn't seem to work at the time so I thought it had to do with the month being in all caps. I probably didn't restart my API :man_facepalming:.

Reply:
46c624a

Reply:
Use nodemon!